### PR TITLE
Adding enable naive CTU analyzer option to analyze-build.

### PR DESCRIPTION
**Pull request by szepet**
This option was added to the Static Analyzer in 3558c29dffdbd1eabf0cb17d9b4e532c625e5595 , however, the running script was not updated.

### CODE REVIEW

File: tools/scan-build-py/libscanbuild/analyze.py [hunk 1]
```diff
@@ -639,7 +639,8 @@ def dispatch_ctu(opts, continuation=run_analyzer):
                 + opts['flags'] + [opts['file']]
             triarch = get_triple_arch(cmd, cwd)
             ctu_options = ['ctu-dir=' + os.path.join(ctu_config.dir, triarch),
-                           'reanalyze-ctu-visited=true']
```
Comment (Xazax-hun) [inline tools/scan-build-py/libscanbuild/analyze.py]: Do we need this reanalyze flag at all? As far as I remember we no longer have this.

Comment (gerazo) [reply tools/scan-build-py/libscanbuild/analyze.py]: I think it was removed, so we don't.

File: tools/scan-build-py/libscanbuild/analyze.py [hunk 2]
```diff
@@ -639,7 +639,8 @@ def dispatch_ctu(opts, continuation=run_analyzer):
...
+                           'reanalyze-ctu-visited=true',
+                           'experimental-enable-naive-ctu-analysis=true']
```
Comment (gerazo) [inline tools/scan-build-py/libscanbuild/analyze.py]: Are we sure that setting it to constant on is what we want? Was it tested accordingly?

Comment (szepet) [reply tools/scan-build-py/libscanbuild/analyze.py]: this is set only when ctu_config.analyze is set. Test files were updated but they call clang directly so this parameter was added to the line containing the lit running command.

Comment (gerazo) [reply tools/scan-build-py/libscanbuild/analyze.py]: Ok, so this means that this experimental mode will be the default for everybody running CTU with scan-build-py

**Pull request merged**